.github/workflows/create-gh-issue-and-slack-message.yml

```diff
@@ -0,0 +1,40 @@
+name: Create-Notifications
+
+# Controls when the workflow will run
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_call:
+
+jobs:
+  Create-Notifications:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+
+      - name: Create Github and Slack notifications
+        run: |
+          response=$(curl -sL \
+              -X POST \
+              -H "Accept: application/vnd.github.v3+json" \
+              -H "Authorization: Bearer ${{ secrets.GH_TOKEN }}" \
+              -H "X-GitHub-Api-Version: 2022-11-28" \
+              https://api.github.com/repos/${{ github.repository }}/issues \
+              -d '{"title":"Release Pipeline failed","body":"For details see: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}","labels":["ci-failure"]}')
```

Comment:
This is looking good, but I think we need inputs for the title, maybe labels too. You can set defaults for the values with what you have hardcoded here.